pkgs/development/libraries/gmp/5.1.x.nix

```diff
@@ -1,4 +1,7 @@
-{ stdenv, fetchurl, m4, cxx ? true, withStatic ? true }:
```

Comment:
I think this should have been `withStatic ? false` all along, without a comment to justify it.